project_2/include/express_pass.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "packet.hpp"
+
+#include <random>
+
+class ExpressPass {
+public:
+    const static std::uint64_t min_packet_size = 84 * 8;
+    const static std::uint64_t default_data_size = 1024 * 8;
+    const static std::uint64_t target_link_speed = 5;
+    const static std::uint64_t default_credit_rate_limit_ns = (default_data_size + target_link_speed - 1) / target_link_speed;
+    const static std::uint64_t default_inter_credit_gap_ns = (min_packet_size + default_data_size + target_link_speed - 1) / target_link_speed;
+    const static std::uint64_t min_jitter = (default_inter_credit_gap_ns + 99) / 100;
+    const static std::uint64_t max_jitter = min_jitter * 8;
+
+    ExpressPass() = default;
+    std::uint64_t getRandomJitter() const;
```

Comment:
```suggestion
    std::uint64_t GetRandomJitter() const;
```

Reply:
I think, with lower case `g`  for getter function `getSomething` is more common, so I would leave this as it is.